### DISCUSSION TITLE
Amortized data vs Total Cost data

### DISCUSSION THREAD

**Discussion by estimpfle** [microsoft/finops-toolkit#1113] [Q&A] [answered]
The FINOPS template does not work for my organization and I'm preparing to convert the data myself.  
In Azure you get two data sets Amortized data (EffecticeCosts) and Total Costs= (BilledCosts).  My question is about how does the FOCS merge these two data sets.  Should I start with the Total cost data and bring in the Effectice cost column. Or do I start with the Total costs data set and bring in the column of data for BilledCosts.

**Comment by flanakin** [comment, marked as answer]
@estimpfle Have you tried FOCUS exports? That should give you what you're looking for. We hope you can avoid converting data manually, but if you need to, please use this [FOCUS data conversion instructions](https://learn.microsoft.com/cloud-computing/finops/focus/convert).
